src/ExpressionEvaluator/Core/Source/ExpressionCompiler/DkmUtilities.cs

```diff
@@ -43,19 +43,25 @@ private static IEnumerable<DkmClrModuleInstance> GetModulesInAppDomain(this DkmC
                 });
         }
 
-        internal unsafe static ImmutableArray<MetadataBlock> GetMetadataBlocks(this DkmClrRuntimeInstance runtime, DkmClrAppDomain appDomain)
+        internal static ImmutableArray<MetadataBlock> GetMetadataBlocks(this DkmClrRuntimeInstance runtime, DkmClrAppDomain appDomain)
+        {
+            return runtime.GetMetadataBlocks(appDomain, ImmutableArray<MetadataBlock>.Empty);
```

Comment:
Agree. Thank you!

---
In reply to: [127110805](https://github.com/dotnet/roslyn/pull/20834#discussion_r127110805) [](ancestors = 127110805)